README.md

```diff
@@ -75,10 +75,12 @@ To build with the MSRV you will need to pin `serde` (if you have the feature ena
 
 ```
 # serde 1.0.157 uses syn 2.0 which requires edition 2021
+cargo update -p serde_json --precise 1.0.99
 cargo update -p serde --precise 1.0.156
-cargo update -p half --precise 1.7.1
```

Comment:
We still pin `half` in the fuzz tests.

Reply:
This was a mistake on my part for not confirming ahead of time, but I'm not sure why these deps need to be pinned.  @apoelstra mentioned they where out of sync now with contrib/test.sh https://github.com/rust-bitcoin/rust-bitcoin/issues/1969.

Reply:
I think it's probably fine not to mention `half` since downstream users won't need to pin it.

Re "not sure why", try compiling with 1.48.0 without the pin, and see what happens.

Reply:
I tested building rust-bitcoin as well as a downstream crate that depends on rust-bitcoin without the pinning and both built without a problem using `cargo +1.48 build`.  To build rust-bitcoin, I had to copy `Cargo-recent.lock` -> `Cargo.lock`.  I tested the builds inside a clean bsd jail to be sure there wasn't any cargo cache or something like that.

Reply:
also note the downstream crate is pointing at the master branch on git that I tested.

Reply:
If you are using `Cargo-recent.lock` then you're not testing the pins.

For my part I can't build with 1.48.0 with the current set of pins, because proc-macros2 doesn't build with the latest MSRV. When I add that pin, I then can't build because of some incompatibility with `hashes` macros.

Reply:
After a clean install the pins in `./bitcoin/contrib/test.sh` _without_ changing `Cargo.lock`:

```
cargo update -p serde_json --precise 1.0.99
cargo update -p serde --precise 1.0.156
```

however this does not build for me.

Reply:
> If you are using Cargo-recent.lock then you're not testing the pins.

If that's the case, then do the pins actually need to be run in `./bitcoin/contrib/test.sh`?

Reply:
I tested taking the pins out of test.sh and the test suits passes.